db/seeds.rb

```diff
@@ -0,0 +1,19 @@
+module Shop
```

Comment:
OK. I knew this isn't exactly what seed is for, but I wanted do 'pseudo' seed to imitate having a DB. I will think about moving constants back and seeding them with data from here.